### DISCUSSION TITLE
DNS problem

### DISCUSSION THREAD

**Discussion by badblli** [vercel/community#3618] [Help]
### Summary

I'm getting a 'site unreachable' error on my website only on mobile. This issue occurs when pinging 76.76.21.21 despite updating my dns records

### Steps to Reproduce

I updated the dns settings of my domain on Natro with vercel nameservers, but although the records were successfully updated, my website works successfully only on the desktop. When I try to access the site with mobile internet from my mobile device, I get 'This site cannot be reached' error. When pinging 76.76.21.21, it says the site cannot be reached. My location is Turkey and I am having this problem. When I use VPN, the site opens successfully.

what is the reason for this error? I would appreciate your help because this is an important mistake.

my website address: [](https://www.depoantalya.com/)
note: i am using vercel hobby plan

**Comment by smaeda-ks** [comment]
@badblli Are you still having the issue? It seems your ISP (mobile carrier) might be blocking our IPs for some reason. Can you run these commands from that network (e.g., run on laptop via tethering) and share that with us?
https://gist.github.com/smaeda-ks/5997018d5fe5dce44d047299829288e1
